### PR TITLE
Migration of International > Taxes > Tax rules page

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Migration of International > Taxes > Tax rules page its actions and filters  to symfony
| Type?         | refacto
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | https://github.com/PrestaShop/PrestaShop/issues/10609
| How to test?  | International > Taxes > Tax rule page should work as previously this PR, accessible via `/improve/international/tax_rules_groups/`

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/15375)

### CODE REVIEW

**Comment by matks** [comment]
@RaimondasSapola I'm sorry, I tried to rebase this PR but it seems I have not the required github rights to force-push your PR 😢 so you must do the rebase

File: src/Adapter/TaxRulesGroup/AbstractTaxRulesGroupHandler.php [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+     * Toggles legacy tax rules group status
+     *
+     * @param TaxRulesGroup $taxRulesGroup
+     * @param bool $newStatus
+     *
+     * @return bool
+     *
+     * @throws TaxRulesGroupException
+     */
+    protected function toggleTaxRulesGroupStatus(TaxRulesGroup $taxRulesGroup, bool $newStatus)
```
Comment (Progi1984) [inline src/Adapter/TaxRulesGroup/AbstractTaxRulesGroupHandler.php]:
```suggestion
    protected function setTaxRulesGroupStatus(TaxRulesGroup $taxRulesGroup, bool $newStatus)
```

File: src/Adapter/TaxRulesGroup/AbstractTaxRulesGroupHandler.php [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+            return $taxRulesGroup->delete();
+        } catch (PrestaShopException $e) {
+            throw new CannotDeleteTaxRulesGroupException(
+                sprintf('An error occurred when deleting tax rules group object with id "%s".', $taxRulesGroup->id)
+            );
+        }
+    }
+
+    /**
+     * Toggles legacy tax rules group status
```
Comment (Progi1984) [inline src/Adapter/TaxRulesGroup/AbstractTaxRulesGroupHandler.php]:
```suggestion
     * Set legacy tax rules group status
```

File: src/Core/Domain/TaxRulesGroup/Command/BulkToggleTaxRulesGroupStatusCommand.php [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+namespace PrestaShop\PrestaShop\Core\Domain\TaxRulesGroup\Command;
+
+use PrestaShop\PrestaShop\Core\Domain\TaxRulesGroup\Exception\TaxRulesGroupConstraintException;
+use PrestaShop\PrestaShop\Core\Domain\TaxRulesGroup\ValueObject\TaxRulesGroupId;
+
+/**
+ * Command responsible for multiple tax rules groups status toggling
+ */
+class BulkToggleTaxRulesGroupStatusCommand
```
Comment (Progi1984) [inline src/Core/Domain/TaxRulesGroup/Command/BulkToggleTaxRulesGroupStatusCommand.php]:
```suggestion
class BulkSetTaxRulesGroupStatusCommand
```

**Review by Progi1984 — COMMENTED** [review]
Some naming

**Review by Progi1984 — APPROVED** [review]

**Pull request merged**